airbyte-cdk/python/unit_tests/sources/file_based/file_types/test_avro_parser.py

```diff
@@ -0,0 +1,137 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import pytest
+from airbyte_cdk.sources.file_based.file_types import AvroParser
+
+
+@pytest.mark.parametrize(
+    "avro_type, expected_json_type, expected_error",
+    [
+        # Primitive types
+        pytest.param("null", {"type": "null"}, None, id="test_null"),
```

Comment:
If we add handling for the types mentioned in my other comment we'll want to add tests.